### PR TITLE
Replace `256` with `layout.TileWidth`/`layout.EntryBundleWidth`

### PR DESCRIPTION
**Pull request by roger2hk**

### CODE REVIEW

File: cmd/conformance/posix/main.go [hunk 1]
```diff
@@ -58,11 +59,11 @@ func main() {
 	s, a := getSignersOrDie()
 
 	// Create the Tessera POSIX storage, using the directory from the --storage_dir flag
-	storage, err := posix.New(ctx, *storageDir, *initialise, tessera.WithCheckpointSigner(s, a...), tessera.WithBatching(256, time.Second))
+	storage, err := posix.New(ctx, *storageDir, *initialise, tessera.WithCheckpointSigner(s, a...), tessera.WithBatching(layout.TileWidth, time.Second))
```
Comment (AlCutter) [inline cmd/conformance/posix/main.go:62]: I wouldn't replace this one; 256 here is kinda arbitrary, and I'd be worried about accidentally creating some "guideline" about how this number must be chosen.

File: log.go [hunk 2]
```diff
@@ -21,14 +21,15 @@ import (
...
+	"github.com/transparency-dev/trillian-tessera/api/layout"
 	"github.com/transparency-dev/trillian-tessera/internal/options"
 	"golang.org/x/mod/sumdb/note"
 	"k8s.io/klog/v2"
 )
 
 const (
 	// DefaultBatchMaxSize is used by storage implementations if no WithBatching option is provided when instantiating it.
-	DefaultBatchMaxSize = 256
+	DefaultBatchMaxSize = layout.TileWidth
```
Comment (AlCutter) [inline log.go:32]: Nor here

**Review by AlCutter — APPROVED** [review]
Nice one, thanks @roger2hk for doing this!

LGTM, just with a suggestion to revert the lines in here which are related to batching & dedup.

File: cmd/conformance/posix/main.go (hunk 1 above)
Comment (roger2hk) [reply cmd/conformance/posix/main.go]: Done.

File: cmd/conformance/posix/main.go [hunk 3]
```diff
@@ -58,11 +59,11 @@ func main() {
...
 	if err != nil {
 		klog.Exitf("Failed to construct storage: %v", err)
 	}
-	dedupeAdd := tessera.InMemoryDedupe(storage.Add, 256)
+	dedupeAdd := tessera.InMemoryDedupe(storage.Add, layout.TileWidth)
```
[the remark this answers is not in the document]
Comment (roger2hk) [reply cmd/conformance/posix/main.go]: Done.

File: log.go (hunk 2 above)
Comment (roger2hk) [reply log.go]: Done.

**Pull request merged**